even-more-python-for-beginners-data-tools/15 - Visualizing data with Matplotlib/15 - Visualizing correlations.ipynb

```diff
@@ -43,9 +39,7 @@
   },
   {
    "cell_type": "markdown",
-   "execution_count": null,
```

Comment:
@devquint Why was this line deleted?

Reply:
There might be some environment-related things on jupyter notebook.